verification/curator-service/api/src/controllers/sources.ts

```diff
@@ -195,6 +201,7 @@ export default class SourcesController {
                 source.toAwsStatementId(),
             );
             source.set('automation.schedule.awsRuleArn', createdRuleArn);
+            await this.sendNotifications(source, NotificationType.Create);
```

Comment:
up to you but I wouldn't feel super bad if we were not waiting for the emails to be sent.

Reply:
Good point -- we probably don't need to wait. For now it might be nice, though, to simplify making sure they're working.